server/bin/basket-proxy-server.js

```diff
@@ -123,6 +124,7 @@ function basketRequest(path, method, params, done) {
     url: API_URL + path,
     strictSSL: true,
     method: method,
+    timeout: API_TIMEOUT,
```

Comment:
Aye, that's what I was worried about - playing around with some local timeouts now to see if I can figure out the necessary magic